spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSkipperStreamService.java

```diff
@@ -88,6 +91,7 @@ public DefaultSkipperStreamService(StreamDefinitionRepository streamDefinitionRe
 		Assert.notNull(appDeploymentRequestCreator, "AppDeploymentRequestCreator must not be null");
 		this.skipperStreamDeployer = skipperStreamDeployer;
 		this.appDeploymentRequestCreator = appDeploymentRequestCreator;
+		this.argumentSanitizer = new ArgumentSanitizer();
```

Comment:
Do we still need `ArgumentSanitizer` in this class again as `AuditServiceUtils` already delegates that?

Reply:
I see the reasoning as we explicitly call the `sanitizeProperties` in this class. I can try to move add this in `AuditServiceUtils` and refactor this.